src/test/llmnode_test.cpp

```diff
@@ -63,6 +63,47 @@
 
 using namespace ovms;
 
+class EmbeddingsFlowHttpTest : public ::testing::Test {
```

Comment:
/src/test/mediapipeflow_test.cpp should be extended to test the presence of new calculator